v2/stacks/auth/README.md

```diff
@@ -25,14 +28,35 @@ AWS_SESSION_TOKEN
 SERVICE_AUTH_TOKEN
 
 # You'll also need your zebedee root for content
+# E.g. zebedee_root=/Users/your-name/zebedee-content
 zebedee_root
+
+# You may also need these AWS settings. The profile should point to sandbox.
+AWS_PROFILE
+AWS_REGION
+
+# You may also need the following other environment variables. Get the values from the relevant secrets in sandbox:
+
+# WEBSITE and PUBLISHING_THREAD_POOL_SIZE and TRANSACTION_STORE from the-train
+WEBSITE
+PUBLISHING_THREAD_POOL_SIZE
```

Comment:
I thought these two were not needed in the end?

Reply:
To be honest I couldn't remember if they were needed or not but I've just tried it without them in local env and it still works fine so I've removed them now 👍